.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,42 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+  "name": "Node.js & TypeScript",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  "image": "mcr.microsoft.com/devcontainers/typescript-node:0-16",
+  "features": {
+    "ghcr.io/devcontainers/features/github-cli:1": {},
+    "ghcr.io/devcontainers/features/rust:1": {},
+    "devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {}
+  },
+
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  // "features": {},
+
+  // Use 'forwardPorts' to make a list of ports inside the container available locally.
+  // "forwardPorts": [],
+
+  // Use 'postCreateCommand' to run commands after the container is created.
+  "onCreateCommand": "npm install -g @microsoft/rush @rushstack/heft && rush update --bypass-policy",
```

Comment:
```suggestion
  "onCreateCommand": "npm install -g @microsoft/rush @rushstack/heft && rush install --bypass-policy",
```

Reply:
Can this be updated to point to a BASH script instead of the commands chained with `&&`? Can we also see if there's a way to get at the GitHub username and display name during setup to preemptively set the git `user.name` and `user.email` during intialization?

Reply:
I was able to find a github API endpoint which can give me a full user name based on their github username! So I made a tiny node script for doing an https call to grab that data and then programmatically set the git config settings.